07. Files/01_ContactBook/01_ContactBook.py

```diff
@@ -0,0 +1,108 @@
+# 1. Use the file as a database to store phonebook entries from previous tasks.
+#
+# Your phonebook, previously contained in a dict, should be stored as text in JSON format.
+# When closing the program and opening it again, all previous data should be available.
+
+import json
+import os
+
+
+with open("contacts.json", "a+") as file:
+    is_empty = os.stat("contacts.json").st_size == 0
+    if is_empty:
+        file.write("{}")
+
+
+def add_contact():
+    with open("contacts.json", "r+") as file_data:
+        our_contacts = json.loads(file_data.read())
+
+    with open("contacts.json", "w") as file_data:
```

Comment:
nit: your file is already open using `r+` so you can move to the start of the file using `seek` and write to it and not open it again.